pkg/monitoring/alloy/templates/alloy-config.alloy.template

```diff
@@ -44,9 +44,10 @@ prometheus.remote_write "default" {
       insecure_skip_verify = {{ .RemoteWriteTLSInsecureSkipVerify }}
     }
     queue_config {
-      capacity = {{ .QueueConfigCapacity }}
+      capacity             = {{ .QueueConfigCapacity }}
+      max_shards           = {{ .QueueConfigMaxShards }}
       max_samples_per_send = {{ .QueueConfigMaxSamplesPerSend }}
-      max_shards = {{ .QueueConfigMaxShards }}
+      sample_age_limit     = {{ .QueueConfigSampleAgeLimit }}
```

Comment:
Adding it to alloy seems like a no-brainer, because our alloy clusters are quite recent.
But for prometheus: are we sure all agents currently in use support this setting? Quoting you on slack: "maybe it's recent" :stuck_out_tongue_closed_eyes:

Reply:
That's why it's only for alloy and not thé agent :)